### ISSUE TITLE
backtick on french azerty keyboard is bugged

### ISSUE THREAD

**Issue by kud** [michaelgmcd/vscode-language-babel#77]
![screenshot-2020-08-18-11-47-58](https://user-images.githubusercontent.com/655838/90498521-bba0e480-e148-11ea-98c6-75ccf30e63f3.gif)

**Comment by michaelgmcd** [comment]
After an initial glance, this doesn't seem related to the French keyboard. The backtick character doesn't seem to be auto-closing properly.

**Comment by kud** [comment]
Yes, you need to use the touch "space" just after. This bug doesn't happen with the initial vs code javascript language.
